index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/home/safeez/Desktop/Portfolio/Portfolio-Website/style.css">
+    <title>Portfolio-Website</title>
+</head>
+<body>
+    <div class="header container">
+        <div class="header one">My Logo</div>
+
+        <div class="header menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </div>
+    </div>
```

Comment:
- [ ] Please use a header tag in place of a div tag in wrapping this section for semantic reasons. This will also improve the accessibility of your site.
- [ ] The logo should be wrapped in an anchor(a) tag, so it can be linked to the homepage when clicked
- [ ] The hamburger button should be wrapped in a nav tag for semantic purposes
- [ ] I can see you display this section with flex but there is no space between child elements
![image](https://user-images.githubusercontent.com/43995830/153550360-cc9ce6a6-05e6-4fdb-83de-4ee7fa118c13.png)